rpcs3/Emu/Cell/lv2/sys_spu.cpp

```diff
@@ -733,7 +733,7 @@ error_code sys_spu_thread_initialize(ppu_thread& ppu, vm::ptr<u32> thread, u32 g
 	{
 		if (g_cfg.core.max_spurs_threads < 6 && group->max_num > 0u + g_cfg.core.max_spurs_threads)
 		{
-			if (group->name.ends_with("CellSpursKernelGroup"))
+			if (group->name.ends_with("CellSpursKernelGroup") && !group->name.starts_with("_libsail"))
```

Comment:
Seems like the underscore in "_libsail" is there to space for a user-provided name prefix.
So this should be more accurate:

```suggestion
			if (group->name.ends_with("CellSpursKernelGroup") && !group->name.ends_with("_libsailCellSpursKernelGroup"))
```

Reply:
Changed, thanks